data/json/mutations.json

```diff
@@ -3913,7 +3913,8 @@
         "description" : "Your skin is rather pale.",
         "changes_to" : ["ALBINO"],
         "leads_to" : ["TROGLO"],
-        "category" : ["LUPINE"]
+        "category": [ "LUPINE" ],
+        "types": [ "skin_tone" ]
     },{
```

Comment:
Remove ``PALE`` with all references from here and move it to cosmetic mutations, it will simplify a lot of things. There is even an open issue about it somewhere.

Reply:
Pale is so far defined as a mutation, with visibility and ugliness stats and everything, and is connected to developing further mutations. I'm reluctant to move it around at the moment, particularly as I have no shortage of caucasian skin tones.